packages/office-ui-fabric-react/src/components/ChoiceGroupOption/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './ChoiceGroupOption';
+export * from './ChoiceGroupOption.types';
```

Comment:
why is `ChoiceGroupOption` at components level? can this be used as an independent component outside of `ChoiceGroup`? if not, can you put it under ChoiceGroup folder?

Reply:
I think there is value in letting this be used as an independent component, though that might be good to do at a later date once everyone is happy with this current abstraction.